Information governance capabilities (#86)
sa-tre/satre-specification: docs/source/standard.md

### 1.1 Compliance, monitoring and reporting

*The ability of the TRE organisation to monitor compliance with internal and external requirements, agreements, laws and standards.*

| Statement | Guidance | Mandatory status |
| --------- | -------  | --- |

> | You are able to audit your TRE organisation against relevant requirements and standards | If you are publicly accredited against a standard, for instance ISO27001, DSPT, CE+ or the SATRE specification, you must have processes in place to ensure you remain compliant | Mandatory |

Reviewer: ```suggestion
| You are able to audit your TRE organisation against relevant requirements and standards | If you are publicly accredited against a standard, for instance ISO27001, DSPT, CE+, etc, you must have processes in place to ensure you remain compliant | Mandatory |
```

> | You are able to audit your TRE organisation against relevant requirements and standards | If you are publicly accredited against a standard, for instance ISO27001, DSPT, CE+ or the SATRE specification, you must have processes in place to ensure you remain compliant | Mandatory |

Reviewer 2: Who is "you" here? A person? An organisation?

### 1.2 Policy regulation and management

*How an organsation determines what policies and regulations are required and ensures alignment to changes in requirements.*

| Statement | Guidance | Manditory Status
| --------- | -------- | ----------- |
| You have a process in place to ensure any new project requiring a TRE meets relevant legal, ethical and contractual requirements | For example national legislation such as GDPR, discipline specific regulation like GCP or contractural requirements from a specific data provider such as a company or research partner organisation | Mandatory
| You have a process in place to monitor changes to any legal, ethical and contractual requirements, and to update your policies accordingly | | Mandatory |

### 1.3 Quality management

*The ability of the TRE organisation to measure and control quality of processes, documentation and outputs.*

#### Document management

| Statement | Guidance | Mandatory status |
| --------- | -------- | ---------------- |

> | All policies & standard operating procedures relevant to the TRE organisation are controlled | This may include measures like restricting edit access to relevant documents, and recording acceptance of policies for all TRE organisation members | Mandatory |

Reviewer 2: Do we also need some requirement to have any of the following?:
- up-to-date
- well-understood by users/operators
- actually followed
- version controlled
- codified change process

Author: I agree explicitly for version control and codified change process, will add

#### Issue management

| Statement | Guidance | Mandatory status |
| --------- | -------- | ---------------- |
| You have a clear process in place for addressing activity within your TRE organisation that deviates from your policies and standard operating procedures | This can include measures like triage analysis and a process for updating policies | Mandatory |

> | You have methods in place to record progress in resolving issues with, and deviations against, your policies | | Mandatory |

Reviewer 2: What's the difference between these two? The first one is people doing something wrong and the second one isn't?

Author: The idea is to have a way to identify issues, then a way of tracking progress for resolving them

### 1.4 Risk management

*The ability of the TRE organisation to measure, forecast and evaluate risks to information.*

#### Risk assessment

| Statement | Guidance | Mandatory Status |
| --------- | -------- | ---------------- |
| You have a way to score risk to understand the underlying severity | You have a risk assesment methodology for scoring risks on multiple axes such as impact and likelihood | Mandatory |

> | You have a process for treating risk using additional controls | Risks can be reduced to a level which brings it within agreed levels of appetite | Mandatory |

Reviewer: ```suggestion
| You have a process for mitigating risk using additional controls | Risks can be reduced to a level which brings it within agreed levels of appetite | Mandatory |
```

> | You have an understanding of risk appe*tite | You have an understanding of risk appetite, ownership of risk and ability to accept risk which falls outside of the appetite should that become necessary | Mandatory |

Reviewer: How does this differ from the first two statements?

Author: The key bit seems to be ownership and ability to accept risk outside appetite, not covered by the previous too. Will update wording a little bit

> | You carry out a Data Protection Impact Assessment (DPIA) for all projects requiring a TRE that are working with personal data | a DPIA is a process designed to identify risks arising out of the processing of personal data and to minimise these risks as far and as early as possible | Mandatory |

Reviewer 2: This is very UK-centric. Do we want to be? If not, how about a (non-capitalised) "data processing assessment"? Also, do we only care about personal data? How about "sensitive data" instead?

Reviewer: This was my thought too, but decided not to open that can of worms! I think it'd be great if SATRE could be relevant across Europe, and eventually beyond.

### 1.6 Project management

*The ability of the TRE organisation to manage projects effectively.*

#### Project onboarding

| Statement | Guidance | Mandatory status |
| --------- | -------- | ---------------- |
| You have checks in place to ensure a project has the legal, financial and ethical requirements in place for the duration of the project | This includes checks that contracts are in place where required, adequate funding is available for the duration of the project, and responsibilities concerning data ownership are understood by all parties | Mandatory |

#### Project closure

| Statement | Guidance | Mandatory status |
| --------- | -------- | ---------------- |
| You have standard processes in place for the end of a project, that follow all legal requirements and data security best practice | This includes the archiving of quality and log data along with the archiving or deletion of data sets | Mandatory |

#### Roles and responsibilities

| Statement | Guidance | Mandatory status |
| --------- | -------- | ---------------- |

> | You have clearly defined roles and responsibilities within your TRE organisation for all members | This may include roles such as users, system administrators, system operators, data providers and more. Every member of your TRE organisation should have a pre-defined role with clear powers and responsibilities | Mandatory |

Reviewer: What do you mean by `member`? Does it include e.g. external researchers outside the organisation operating the TRE?

Reviewer 2: There's a separate "Roles" section later. Can we combine these two?

Author: Yeah will need this to be defined in a glossary of terms. It includes anyone associated with the TRE, including internal/external researchers, operators, builders, admins, DPRs and more (catch all term in essence)

### 1.7 Researcher accreditation

The ability of the TRE organisation to ensure that people with access to data are identified correctly and they are suitably qualified.

> #### Onboarding members

Reviewer: This section is under `Researcher accreditation`, but the statements seem to be targetted at all `members`. Is it intended to be specific to researchers?

Reviewer 2: I think "researcher" is an unnecessarily loaded word. People who aren't doing research might want to use a TRE. Can we check preferred naming with the glossary people? Probably just "user" or "end user" would work here.

| Statement | Guidance | Mandatory status |
| --------- | -------- | ---------------- |
| You have clear onboarding processes in place for all roles within your TRE organisation | This may include all members signing role-specific terms of use, and completing role specific training | Mandatory

> | You have a robust method for identifying accredited members of your TRE organisation, prior to their accessing of sensitive data | This may include 2 factor authentication, ID checks or email verification | Mandatory |

Reviewer 2: ```suggestion
| You have a robust method for identifying accredited members of your TRE organisation, prior to their accessing of sensitive data | This may include multi-factor authentication (MFA), ID checks or email/phone verification | Mandatory |
```

#### Training management and delivery

| Statement | Guidance | Mandatory status |
| --------- | -------- | ---------------- |
| You have relevant training for all roles within the TRE organisation, and the ability to deliver this training | This may include: Cyber security training, GDPR training, and higher level training for system operators | Mandatory

> | All TRE organisation members have completed relevant training within the last 12 months | | Mandatory |

Reviewer: How strict is `12 months`? Technically it means you have to complete the training on the same day or earlier in the year each time. Could say 13 months instead?

Reviewer 2: 12 months is a strict requirement for DSPT. I'm not sure it needs to be for us. How about "All TRE organisation members have recently completed relevant training, for example within the last twelve months."

Author: I reckon talking point for today, making it a clear timeframe helps with credibility

> | You have a process in place to monitor all TRE organisation training completions & requirements | This process should document which members have completed which training, when the training was completed, and the date the training expires. It should also document how you will notify members when their training is about to expire, and ensure they do not have access to any TRE if relevant training is over 12 months old | Mandatory |

Reviewer 2: ```suggestion
| You have a process in place to monitor all TRE organisation training completions & requirements | This process should document which members have completed which training, when the training was completed, and the date the training expires. It should also document how you will notify members when their training is about to expire, and ensure they do not have access to any TRE if relevant training is out-of-date | Mandatory |
```

| Statement                                                                                                                                  | Guidance                                                                                                                                                                                             | Manditory Status |
| ------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------- |
| You have a process in place to ensure any new project requiring a TRE meets relevant legal, ethical and contractual requirements           | For example national legislation such as GDPR, discipline specific regulation like GCP or contractural requirements from a specific data provider such as a company or research partner organisation | Mandatory        |

> | You have a process in place to monitor changes to any legal, ethical and contractual requirements, and to update your policies accordingly |                                                                                                                                                                                                      | Mandatory        |

Reviewer 3: Is it the process that is important, or ensuring that the requirements are met?

| ---------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------- |
| You have a way to score risk to understand the underlying severity                                               | You have a risk assesment methodology for scoring risks on multiple axes such as impact and likelihood                                                                           | Mandatory        |
| You have a process for mitigating risk using additional controls                                                 | Risks can be reduced to a level which brings it within agreed levels of appetite                                                                                                 | Mandatory        |

> | You have an understanding of risk appetite                                                                       | This includes understanding ownership of risk, and ability to accept risk which falls outside of the appetite should that become necessary                                       | Mandatory        |

Reviewer 2: Swap these two around?

> | You have an understanding of risk appetite                                                                       | This includes understanding ownership of risk, and ability to accept risk which falls outside of the appetite should that become necessary                                       | Mandatory        |

Reviewer 3: How do we prove this? A risk appetite document or statement?
